odl/contrib/datasets/ct/mayo.py

```diff
@@ -233,25 +236,41 @@ def load_reconstruction(folder, slice_start=0, slice_end=-1):
         data_array = data_array.reshape([cols, rows], order='C')
         data_array = np.rot90(data_array, -1)
 
-        # Convert from CT numbers to densities
-        data_array /= 1024.0
+        # Convert from storage type to densities
+        # TODO: Optimize these computations
+        hu_values = (dataset.RescaleSlope * data_array +
```

Comment:
Now I will reveal my lack of knowledge in this field: what is the difference between this `hu_values` and the `hu_factor` on line 57?